src/compressed_tensors/utils/helpers.py

```diff
@@ -190,7 +195,7 @@ def decorator(func: Callable[[Any], Any]):
 
         @wraps(func)
         def wrapped(*args, **kwargs):
-            warnings.warn(message, DeprecationWarning, stacklevel=2)
+            logger.bind(log_once=True).warning(message)
```

Comment:
I think it's actually preferred to using the `warnings` module for this use case ([see python docs](https://docs.python.org/3/howto/logging.html#when-to-use-logging)). We can do the log once with something like:
```python
logged = False
@wraps(func)
def wrapped(*args, **kwargs):
    nonlocal logged
    if not logged:
        warnings.warn(...)
        logged = True
```
        


```

Reply:
No, it should work as is. Sorry, I didn't include the whole code for simplicity but this is still within the `decorator(func: T)` fn so this should define a different logged variable for each `func` this is called with.

Reply:
That's fine, I've reverted to `warnings.warn(message, DeprecationWarning, stacklevel=2)`